src/jobs/rawnet.go

```diff
@@ -15,104 +16,95 @@ import (
 	"github.com/Arriven/db1000n/src/utils/templates"
 )
 
-// rawNetJobConfig comment for linter
-type rawNetJobConfig struct {
```

Comment:
yeah, but that way we have to return 4 values from the function instead of 2, not sure it's worth it

Reply:
I can pack up 3 return values into a struct if you insist, but it will be a different one (see the return types).

Reply:
Ah, right, didn't notice that you've also split parsing and execution of templates here (I only slept for 3 hours today). Makes sense then.
Grouping these values together would be nice but I've been thinking that it could be even better to generalize these 2 functions into one and avoid all the copy-paste between them. Can be done separately though